src/locales/pl.ts

```diff
@@ -293,7 +296,7 @@ export default {
     martin_de_porres_religious: 'Św. Marcina de Porres, zakonnika',
     martin_i_pope: 'Św. Marcina I, papieża i męczennika',
     martin_of_tours_bishop: 'Św. Marcina z Tours, biskupa',
-    martin_wu_xuesheng_and_companions_martyrs: 'Świętych męczenników Marcina Wu Xueshenga i Towarzyszy',
+    martin_wu_xuesheng_and_companions_martyrs: 'Świętych męczenników Marcina Wu Xueshenga, i Towarzyszy',
```

Comment:
I am not sure of the more recent editions, however, in the [_editio typica_](https://ministranci.waw.pl/wp-content/uploads/docs/Mszal.pdf#page=850), they don’t use a comma before `i Towarzyszy`.